spec/services/membership_modifier_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'rails_helper'
+
+RSpec.describe MembershipModifier do
+  let(:target_joinable) { FactoryGirl.create(:group) }
+  let(:target_user) { FactoryGirl.create(:user) }
+  let(:requesting_user) { FactoryGirl.create(:user) }
+
+  describe '#update_membership' do
+    context 'when requesting_user is not an admin of target_joinable' do
+      it 'does nothing' do
+        expect(Membership).not_to receive(:update)
+
+        MembershipModifier.create_membership(requesting_user: requesting_user,
+                                             joinable: target_joinable,
+                                             target_user: target_user,
+                                             admin: false)
+      end
+    end
+
+    context 'when requesting_user is an admin of target_joinable' do
+      before do
+        FactoryGirl.create(:group_admin_membership, user: requesting_user, joinable: target_joinable)
+      end
+
+      context 'and the target_user is an admin of target_joinable' do
+        before do
+          FactoryGirl.create(:group_admin_membership, user: target_user, joinable: target_joinable)
+        end
+
+        context 'and the admin: false parameter is passed in' do
+          it "changes the target_user's membership to a member of target_joinable" do
+            MembershipModifier.update_membership(requesting_user: requesting_user,
+                                                 joinable: target_joinable,
+                                                 target_user: target_user,
+                                                 admin: false)
+
+            expect(target_joinable.member? target_user).to be_truthy
```

Comment:
How come you're not using `.to be true`? Shouldn't that method just return `true` or `false`